### ISSUE TITLE
Parser Blows Up

### ISSUE THREAD

**Issue by jeffjewiss** [pengwynn/fumblr#2]
I can get Fumblr setup and running fine previewing the demo blog and theme, but as soon as I put my own theme markup in tumblr.html the parser blows up with the following error:

```
NoMethodError at /
undefined method `collect' for nil:NilClass
file: parser.rb location: block in parse line: 307
```

I really want Fumblr to work because it would save a ton of time developing themes having a local environment and being able to use compass/sass.

I know you didn't write the parser, but are you aware of any intricacies of the theme blocks that would cause this?

**Comment by pengwynn** [comment]
Can you gist me your theme?

**Comment by jeffjewiss** [comment]
Normally I would, but it's a paid theme for work so I don't feel comfortable putting the whole theme in a gist. Sorry.

I dug around a bit more last night and after selectively removing parts of the theme I narrowed it down to the tag block that was causing the problem:

```
<div class="tags">
<ul>
    {block:Tags}
    <li><a href="{TagURL}">{Tag}</a></li>
    {/block:Tags}
</ul>
</div>
```

I noticed that on jphasting's Thimblr he lists "Get notes working, day pages and tag pages" as a to-do. So I suppose it's not expected to work, but it probably shouldn't prevent the entire page from rendering.
